src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -88,6 +88,7 @@ public static string PathVariable
         public static string PluginTracePrefix = "##[plugin.trace]";
         public static readonly int AgentDownloadRetryMaxAttempts = 3;
         public const string projectName = "projectName";
+        public const string TaskSDKCommandTokenEnvVar = "TASK_SDK_COMMAND_TOKEN";
```

Comment:
we need a better name for this.  E.g. CorrelationId or LogId instead of 'token'

Reply:
Changed the name to the COMMAND_CORRELATION_ID